code-challenges.js

```diff
@@ -11,26 +28,60 @@ var secretCodeWord1 = "lackadaisical"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
 
-// --------------------2) Create a function that takes in an array and returns all the words that have the letter a in them.
+//TESTING
+console.log(coder(secretCodeWord1))
+console.log(coder(secretCodeWord2))
 
+// --------------------2) Create a function that takes in an array and returns all the words that have the letter a in them.
 
+const ayo = (array) => {
+    newArr = array.filter(value => value.includes("a") || value.includes("A"))
+    return newArr.join(" ")
+}
```

Comment:
yay! you can even write `newArr = array.filter(value => value.toLowerCase().includes("a"))` if you'd like!